docs/ssr.mdx

```diff
@@ -120,7 +120,11 @@ res
 
 ```jsx
 // Hydration of the ids in `data-emotion-css` will automatically occur when the cache is created
-const cache = createCache()
+const cache = createCache({ 
+  // The key must match your corresponding cache name set earlier
```

Comment:
what do you mean by "corresponding cache name set earlier"?

Reply:
In the example docs above the key was set on line 87 - 88 to `const key = 'custom';`. So it has to match whatever somebody set to that. I would assume you could end up with multiple keys for whatever reason, but I could be wrong.